tests/data/doctrine2_fixtures/TestFixture1.php

```diff
@@ -1,6 +1,7 @@
 <?php
 
 use Doctrine\Common\DataFixtures\FixtureInterface;
+use Doctrine\Persistence\ObjectManager;
 
 if (version_compare(PHP_VERSION, '7.1', '>')) {
```

Comment:
The main branch of this repository already uses PHP 7.1 as a minimum, so this compatibility logic can be removed, could you please remove this and look for logic like this that needs to be removed?

Reply:
I had to bring this back. Actually, the if is for PHP 7.1 compatibility.
I slightly changed it to `if (PHP_VERSION_ID < 70200)`.